model/symflower/symflower.go

```diff
@@ -48,9 +51,42 @@ func (m *Model) GenerateTestsForFile(logger *log.Logger, language language.Langu
 		return nil, pkgerrors.WithStack(err)
 	}
 
+	characterCount, err := countCharactersOfGeneratedFiles(repositoryPath, extractGeneratedFilePaths(output))
+	if err != nil {
+		return nil, err
+	}
+
 	return metrics.Assessments{ // Symflower always generates just source code when it does not fail, so no need to check the assessment properties.
-		metrics.AssessmentKeyResponseNoExcess: 1,
-		metrics.AssessmentKeyResponseWithCode: 1,
-		metrics.AssessmentKeyProcessingTime:   uint64(time.Since(start).Milliseconds()),
+		metrics.AssessmentKeyProcessingTime:                     uint64(time.Since(start).Milliseconds()),
```

Comment:
This needs to go right after the call to `CommandWithResult` and **before** the call to `countCharacters...`, otherwise we will measure the character counting as processing time.  I.e.:

```go
output, err := util.CommandWithResult(...
if err ... {
}

processingTime := time.Since(start)

characterCount, err := countCharactersOfGeneratedFiles(...
```